C/charon/bundled/patches/m_pi.patch

```diff
@@ -0,0 +1,34 @@
+commit d6da81493b465a47436c4f6bf6b4abf3c8502dac
+Author: Keno Fischer <keno@juliacomputing.com>
+Date:   Sun Oct 15 01:37:16 2023 +0000
+
+    Add defines for M_PI
+
+diff --git a/src/Charon_FreqDom_Parameters.hpp b/src/Charon_FreqDom_Parameters.hpp
+index 4711883..3833b99 100644
+--- a/src/Charon_FreqDom_Parameters.hpp
++++ b/src/Charon_FreqDom_Parameters.hpp
+@@ -17,6 +17,9 @@
+ #include "Teuchos_ParameterList.hpp"
+ #include "Teuchos_RCP.hpp"
+ #include "Teuchos_ArrayRCP.hpp"
++#ifndef M_PI
```

Comment:
On Win32 you need to define `_USE_MATH_DEFINES` before importing `math.h` to make the constants visible: https://learn.microsoft.com/en-us/cpp/c-runtime-library/math-constants?view=msvc-170

Reply:
I tried that, but it didn't work. Is there a mingw difference? I can try again.

Reply:
Pretty sure it should work, because we use it in a few places: https://github.com/search?q=repo%3AJuliaPackaging%2FYggdrasil%20_USE_MATH_DEFINES